eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/searchlist/resultupdates/ClearNoMatterOfQuery.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.searchlist.resultupdates;
+
+import com.schedjoules.eventdiscovery.framework.list.changes.nonnotifying.ClearAll;
+import com.schedjoules.eventdiscovery.framework.list.changes.nonnotifying.NonNotifyingChangeableList;
+
+
+/**
+ * {@link ResultUpdate} to clear all items regardless of the current query.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class ClearNoMatterOfQuery<T> implements ResultUpdate<T>
```

Comment:
I would call this `ForcedClear`, seems a bit more readable. Same for `ShowSingleNoMatterOfQuery` which I would call `ForcedShowSingle`.